docs/how-to/enable-intel-gpu.rst

```diff
@@ -0,0 +1,124 @@
+.. _enable_intel_gpu:
+
+Enable Intel GPUs
+=============================
+
+This guide describes how to configure DSS to utilise the Intel GPUs on your machine. This can be done by enabling the Intel device plugin on your MicroK8s cluster.
+
+Prerequisites
+-------------
+
+* :ref:`DSS CLI is installed <install_DSS_CLI>` and :ref:`initialised <initialise_DSS>`.
+* Your machine has an Intel GPU.
+
+Install the Intel GPU plugin
+----------------------------
+
+To ensure DSS can utilise Intel GPUs:
+1. The Intel GPU drivers must be installed.
+2. MicroK8s must be set up to utilise the Intel GPU drivers.   
```

Comment:
Can we add a link for further info to this point?

Reply:
I will add hyperlinks on the sections

Reply:
Because I changed the structure of each sections, this list is not longer needed